amplify/backend/ts-code/src/api/GetItem.ts

```diff
@@ -32,31 +32,36 @@ export class GetItem {
             scratch.key = request
             return this.transactionsTable.delete(number)
                     .then(() => this.execute(scratch))
+                    .then((entry: MainSchema) => {
+                        let returnString = header(entry)
+                        Object.keys(entry.items).forEach((id: string) => returnString += item(entry, id))
+                        return returnString
+                    })
```

Comment:
Is there anyway to rewrite this with async awaits? .then chaining gets messy pretty quickly imo.

Reply:
Dk what is js convention, but I would rather chain promises, and never use awaits. You can pass a promise to AWS Lambda, and it will take care of it.

Reply:
There's probably no strict JS convention, but I think most people prefer async await over promise chaining these days. If that's how AWS is built, then that's fine.